src/frontend/src/app/startpage/startpage.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="iconcontainer">
+  <button mat-fab class="share">
+    <mat-icon>share</mat-icon>
+  </button>
+</div>
+
+<div class="center">
+  <pre><h1 style="font-size:400%"><b>
+
+Welcome to CodeChamp!
+</b></h1></pre>
+</div>
+
+<div class="buttoncontainer">
+  <div
+    style="
+      position: absolute;
+      opacity: 1;
+      transform: translateX(0%) translateZ(0px);
+    "
+  >
+    <div class="roundbutton" [routerLink]="['/problem']">Start Game</div>
+    <div
+      class="roundbutton"
+      matTooltip="Coming soon!"
+      matTooltipPosition="after"
+    >
+      Host room
+    </div>
+    <div
+      class="loginbutton"
+      matTooltip="Coming soon!"
+      matTooltipPosition="after"
+    >
+      <div class="loginicon">
```

Comment:
icon and right arrow seems misplaced in the image, should fix before demo

Reply:
![image](https://user-images.githubusercontent.com/25267171/202925911-6b3bfff8-4c15-4a6a-9263-61c070c69e10.png)